src/posix/platform/netif.cpp

```diff
@@ -1170,12 +1170,10 @@ static void processNetifAddrEvent(otInstance *aInstance, struct rt_msghdr *rtm)
     struct sockaddr_in6 addr6;
     struct sockaddr_in6 netmask;
     uint8_t *           addrbuf;
-    unsigned int        addrlen;
```

Comment:
Is it related change?

Reply:
It doesn't build on my Mac.

Reply:
Seems better to use a separate PR to fix macOS build issues. 

Thoughts?

Reply:
Reverted.